src/test/java/com/commercetools/project/sync/SyncerFactoryTest.java

```diff
@@ -453,22 +470,18 @@ void sync_AsProductsFullSync_ShouldBuildSyncerAndExecuteSync() {
         .haveExactly(1, statisticsLog);
 
     assertThat(productSyncerTestLogger.getAllLoggingEvents())
-        .hasSize(2)
```

Comment:
Looks like they were asserting different things 🤔

Reply:
Yes, they were asserting 2 things but now logger warning message is missing, it is not handled in Java-Sync. Previously it was handled in Project-sync with the warning loggers.